src/SmartComponents/HybridInventoryTabs/ConventionalSystems/PathwaySystems.js

```diff
@@ -0,0 +1,40 @@
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import Inventory from '../../../PresentationalComponents/Inventory/Inventory';
+
+const PathwaySystems = ({ pathway, selectedTags, workloads, SID }) => {
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    return () => {
+      dispatch({
+        type: 'CLEAR_INVENTORY_STORE',
```

Comment:
@mkholjuraev What was the prop of the InventoryTable that made this redundant?

Reply:
It was `initialLoading`, removed this redux clear action. I have verified that the prop [exists](https://github.com/RedHatInsights/insights-advisor-frontend/blob/master/src/PresentationalComponents/Inventory/Inventory.js#L395) in the Inventory component. PTAL